openslides/motions/views.py

```diff
@@ -700,6 +645,30 @@ def create(self, request, *args, **kwargs):
             return Response({'detail': err.message}, status=400)
 
 
+class MotionCommentFieldViewSet(ModelViewSet):
+    """
+    API endpoint for motion comment fields.
+
+    There are the following views: update, partial_update and destroy.
+    """
+    access_permissions = MotionCommentFieldAccessPermissions()
+    queryset = MotionCommentField.objects.all()
+
+    def check_view_permissions(self):
+        """
+        Returns True if the user has required permissions.
+        """
+        print(self.action)
+        if self.action in ('list', 'retrieve'):
+            result = self.get_access_permissions().check_permissions(self.request.user)
+        elif self.action in ('create', 'destroy', 'update'):
```

Comment:
What is about the delete case? Deleting a comment section that is used in any comment is not possible (see on_delete=PROTECT) but we do not want the user to get a HTTP 500 but a HTTP 400.